pornography-hosts

```diff
@@ -7553,6 +7553,23 @@
 127.0.0.1 i0.cdn2a.image.pornhub.phncdn.com
 127.0.0.1 i0.cdn2b.image.pornhub.phncdn.com
 127.0.0.1 i1.cdn2b.image.pornhub.phncdn.com
+adult-empire.com
+apornstories.com
+barepass.com
+creampie-mature.com
+cumilf.com
+grannypornpics.com
+grannyslutphoto.com
+grannyxxxpics.com
+gigapron.com
+hairy-beauty.com
+hd-pornos.net
+leggycash.com
+milfinarium.com
+okporn.com
+scoreland2.com
+xxnx.ink
+xxxvideos.ink
```

Comment:
@beli3ver I can't accept this because the lines above are not in hosts file format.